README.md

```diff
@@ -35,6 +35,6 @@ Following fields are supported:
 
 ## Todo
 
-- [X] Automatic image uploads
-- [ ] Assignees (pick from a list)
+- [x] Automatic image uploads
+- [x] Assignees (pick from a list)
```

Comment:
I was actually thinking this more like mapping the users: so instead of selecting one, get the source list of users and for each of them map to prompt result.

Reply:
Makes sense. Will update